src/Facades/Qr.php

```diff
@@ -64,6 +65,18 @@ public static function getFormSchema(
                         ->statePath($optionsStatePath)
                         ->schema([
                             Hidden::make('type')->default('png'),
+                            Select::make('correction')
```

Comment:
since `correction` is a new option, we need to add it to the `getDefaultOptions` too